macos/FluentUI/Button/Button.swift

```diff
@@ -133,6 +131,10 @@ open class Button: NSButton {
 
 	/// Image to display in the button.
 	override public var image: NSImage? {
+		didSet{
+			/// force image to render as a template image
```

Comment:
i am not sure we should force it. shoudn't client decide?

Reply:
Following the Fluent design, the icon tint color should be the same as the title tint color.

Reply:
but if the button has an avatar image for example, we shouldn't tint it? shouldn't the client try to pass in templated image if they want to be colored?

Reply:
Good point. I think the right fix is to expose the isTemplateImage flag to react native?

Reply:
i do think react native should handle it because if eng want to put random stuff in button, it should jsut respect that. rest of the changes of this PR seems fine.

Reply:
the other thing we could do something like  image?.renderingMode == .automatic ? image?.withRenderingMode(.alwaysTemplate) : image

Reply:
I reverted the template image changes and will let react native handle the custom tinting.